schema/pharmgkbSchema.mcf

```diff
@@ -0,0 +1,333 @@
+Node: dcid:ChemicalCompoundGeneAssociation
+name: "ChemicalCompoundGeneAssociation"
+typeOf: schema:Class
+subClassOf: schema:Intangible
+description: "The association of a chemical compound with a gene."
+
+Node: dcid:compoundID
+name: "compoundID"
+typeOf: schema:Property
+rangeIncludes: dcs:Drug
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:geneID
+name: "geneID"
+typeOf: schema:Property
+rangeIncludes: dcs:Gene
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:pubMedID
+name: "pubMedID"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:isPharmacokineticRelationship
+name: "isPharmacokineticRelationship"
+typeOf: schema:Property
+rangeIncludes: schema:Boolean
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+description: "True if the pair of entities was found in a pharmacokinetic pathway on PharmGKB, or if the Variant Annotation or VIP was annotated with PK in some manner."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:isPharmacodynamicRelationship
+name: "isPharmacodynamicRelationship"
+typeOf: schema:Property
+rangeIncludes: schema:Boolean
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+description: "True if the pair of entities was found in a pharmacodynamic pathway on PharmGKB, or if the Variant Annotation or VIP was annotated with PD in some manner."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:relationshipAssociationType
+name: "relationshipAssociationType"
+typeOf: schema:Property
+rangeIncludes: dcs:RelationshipAssociationTypeEnum
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:relationshipEvidenceType
+name: "relationshipEvidenceType"
+typeOf: schema:Property
+rangeIncludes: dcs:RelationshipEvidenceTypeEnum
+domainIncludes: dcs:ChemicalCompoundGeneAssociation
+
+Node: dcid:pharmGKBID
+name: "pharmGKBID"
+typeOf: schema:Property
+rangeIncludes: dcs:Text
+domainIncludes: dcs:Drug, dcs:ChemicalCompound, dcs:Gene,
+
+Node: dcid:alternateGeneSymbol
+name: "alternateGeneSymbol"
+typeOf: schema:Property
+rangeIncludes: schema:Text
+domainIncludes: dcs:Gene
+
+Node: dcid:alfredID
```

Comment:
Preferably add url: "<link to database>" for all of the IDs as well

Reply:
Unless they were generated using a script in which case the description and url comment don't apply to those schema properties generated by scripts. But let's add those in a separate document rather than adding with manually created schema